src/app/routes/root-routing.module.ts

```diff
@@ -26,7 +26,8 @@ const ROUTE_CONFIG: HtRoute[] = [
                 icon: IconType.Dashboard,
                 label: 'Dashboard'
               },
-              defaultTimeRange: new RelativeTimeRange(new TimeDuration(1, TimeUnit.Hour))
+              defaultTimeRange: new RelativeTimeRange(new TimeDuration(1, TimeUnit.Hour)),
+              shouldSavePageTimeRange: true
```

Comment:
Yeah this crossed my mind. Correct, if the route defines it's own, we'd always want to save. However I'm gonna leave it  as is for readability - to anyone looking at these route configs without the full picture, I think it'd be easier to understand with this property here. One condition being satisfied based on one property rather than two, is a bit clearer